extensions/notebook/src/book/bookModel.ts

```diff
@@ -45,7 +46,7 @@ export class BookModel {
 		if (this.isNotebook) {
 			this.readNotebook();
 		} else {
-			await this.readBookStructure(this.bookPath);
+			await this.readBookStructure(this.bookPath as string);
```

Comment:
Why are you casting to a string? It already is one

Reply:
Initially I was using an object of IBookNotebook or a string for the bookPath, I forgot to remove this one.
Thanks!